src/main/java/com/shopify/model/ShopifyCustomer.java

```diff
@@ -102,14 +95,6 @@ public void setPhone(String phone) {
 		this.phone = phone;
 	}
 
-	public long getOrdersCount() {
```

Comment:
These are only deprecated from the order resources. These will still exist on the /customers endpoint we shouldn't remove these as part of this.  https://shopify.dev/changelog/property-deprecations-in-the-rest-admin-api-order-resource

What do you think if we just made a separate ShopifyOrderCustomer type that is only used as a type on order resources that does not have these fields to avoid the confusion?

Reply:
I think that totally makes sense. I'll implement that and update the PR.

Reply:
Done.